src/VisualStudio/Core/Def/RoslynPackage.cs

```diff
@@ -321,12 +319,6 @@ private void ReportSessionWideTelemetry()
             ComponentModel.GetService<VisualStudioSourceGeneratorTelemetryCollectorWorkspaceServiceFactory>().ReportOtherWorkspaceTelemetry();
         }
 
-        private void DisposeVisualStudioServices()
-        {
-            var referenceManager = this.ComponentModel.GetService<VisualStudioMetadataReferenceManager>();
-            referenceManager.DisconnectFromVisualStudioNativeServices();
```

Comment:
VisualStudioMetadataReferenceManager isn't a normal mef  export, it's a workspace service.  i could attempt to make this work (e.g. getting the VSWorkspace and then getting this service from it), but it seems simpler to just implement IDIsposable on the type itself.

Reply:
Oh excellent. I was wondering why we didn't do this when reviewing the prior PR but figured it wasn't worth the trouble...but now it is. :smile: